src/components/App.js

```diff
@@ -4,8 +4,8 @@ import ButtonPanel from './ButtonPanel';
 
 function App() {
   return (
-    <div className="App">
-      <Display />
+    <div className="App my-5">
```

Comment:
Please use the React.Fragment short syntax as suggested.

Reply:
As you can see I have added class to this, I have added fragments to the Display component, but on this occasion, if I remove this div, I couldn't add class, I ask to approve my project.